akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerWithPings.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.impl.engine.http2
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.ConnectionContext
+import akka.http.scaladsl.model.ContentTypes
+import akka.http.scaladsl.model.HttpEntity
+import akka.http.scaladsl.model.HttpRequest
+import akka.http.scaladsl.model.HttpResponse
+import akka.stream.SystemMaterializer
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+import com.typesafe.config.ConfigFactory
+
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+object Http2ServerWithPings {
+
+  def main(args: Array[String]): Unit = {
+    val config = ConfigFactory.parseString(
+      """
+         akka.loglevel = debug
+         akka.http.server.preview.enable-http2 = on
+         akka.http.server.http2.log-frames = true
+         akka.http.server.http2.ping-interval = 2s
```

Comment:
Would be fine to merge that with `Http2ServerTest`.

Reply:
This was mostly for manual checking with curl while developing, I think we can get rid of it completely now that there is some actual test coverage.